tests/lib/util/npm-util.js

```diff
@@ -66,6 +67,27 @@ describe("npmUtil", function() {
             fs.existsSync.restore();
             fs.readFileSync.restore();
         });
+
+        it("should throw with message when parsing invalid package.json", function() {
+            var logInfo = sinon.stub(log, "info");
```

Comment:
Sure! Not super experienced with `sinon` - would this happen in a `beforeEach`/`afterEach` in the outermost `describe`?